src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,23 @@
-export const TodoInfo = () => {};
+import React from 'react';
+import { TodoType } from '../TodoList';
+import classNames from 'classnames';
+import { UserInfo } from '../UserInfo';
+
+interface Types {
+  todo: TodoType;
+}
+
+export const TodoInfo: React.FC<Types> = ({ todo }) => {
+  return (
+    <article
+      data-id={todo.id}
+      className={classNames('TodoInfo', {
+        'TodoInfo--completed': todo.completed,
+      })}
+    >
+      <h2 className="TodoInfo__title">{todo.title}</h2>
+
+      <UserInfo user={todo.user} />
```

Comment:
user can be null, so let's add the check here
```suggestion
      {todo.user && <UserInfo user={todo.user} />}
```